<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quản Lý Nhân Viên</string>
  </property>
  <property name="styleSheet">
   <string>background-color: #f0f0f0;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- Nhãn tiêu đề -->
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Thông Tin Nhân Viên</string>
    </property>
    <property name="styleSheet">
     <string>font-size: 16px; font-weight: bold;</string>
    </property>
   </widget>
   <!-- Nhãn và trường nhập liệu -->
   <widget class="QLabel" name="lblHoTen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Họ tên:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHoTen">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nhập họ tên</string>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px; padding: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVaiTro">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Vai trò:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboVaiTro">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px; padding: 5px;</string>
    </property>
    <item>
     <property name="text">
      <string>Admin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NhanVien</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblTenDangNhap">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Tên đăng nhập:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTenDangNhap">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nhập tên đăng nhập</string>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px; padding: 5px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMatKhau">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Mật khẩu:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMatKhau">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Nhập mật khẩu</string>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px; padding: 5px;</string>
    </property>
   </widget>
   <!-- Các nút chức năng -->
   <widget class="QPushButton" name="btnThem">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>80</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Thêm</string>
    </property>
    <property name="styleSheet">
     <string>background-color: #4CAF50; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSua">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>80</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Sửa</string>
    </property>
    <property name="styleSheet">
     <string>background-color: #2196F3; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnXoa">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>80</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Xóa</string>
    </property>
    <property name="styleSheet">
     <string>background-color: #f44336; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
   </widget>
   <!-- Tìm kiếm -->
   <widget class="QLabel" name="lblTimKiem">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Tìm kiếm:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTimKiem">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Tìm theo họ tên hoặc tên đăng nhập</string>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px; padding: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTim">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Tìm</string>
    </property>
    <property name="styleSheet">
     <string>background-color: #FF9800; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
   </widget>
   <!-- Bảng nhân viên -->
   <widget class="QTableWidget" name="tableNhanVien">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>860</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; background-color: white;</string>
    </property>
   </widget>
   <!-- Nút thoát -->
   <widget class="QPushButton" name="btnThoat">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
    <property name="styleSheet">
     <string>background-color: #607D8B; color: white; border-radius: 5px; padding: 5px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>